awscli/s3transfer/crt.py

```diff
@@ -176,6 +187,23 @@ def create_s3_crt_client(
     )
 
 
+def _get_crt_throughput_target_gbps(provided_throughput_target_bytes=None):
+    if provided_throughput_target_bytes is None:
+        target_gbps = get_recommended_throughput_target_gbps()
+        logger.debug(
+            'Recommended CRT throughput target in gbps: %s', target_gbps
+        )
+        if target_gbps is None:
+            target_gbps = 10.0
+    else:
+        # NOTE: The GB constant in s3transfer is technically a gibibyte. The
+        # GB constant is not used here because the CRT interprets a gigabyte
+        # as a base power of 10 (i.e. 1000 ** 3 instead of 1024 ** 3).
```

Comment:
utterly trivial
```suggestion
        # NOTE: The GB constant in s3transfer is technically a gibibyte. The
        # GB constant is not used here because the CRT interprets gigabits-for-networking
        # as a base power of 10 (i.e. 1000 ** 3 instead of 1024 ** 3).
```